contracts/test/TestStrategy.sol

```diff
@@ -9,7 +9,15 @@ import {BaseStrategy, StrategyParams} from "../BaseStrategy.sol";
  * for integrators on how to use BaseStrategy
  */
 
+interface VaultLike {
+    function withdraw(uint256 shares, address recipient) external;
+
+    function balanceOf(address holder) external view returns (uint256);
+}
```

Comment:
Could we add this to `VaultAPI` in `BaseStrategy` and import this?

Reply:
fixed in latest